plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -490,7 +544,7 @@ global proc mayaUsdTranslatorExport_SetFromOptions(string $currentOptions, int $
             } else if ($optionBreakDown[0] == "frameSample") {
                 mayaUsdTranslatorExport_SetTextField($optionBreakDown[1], $enable, "frameSampleField");
             } else if ($optionBreakDown[0] == "parentScope") {
-                mayaUsdTranslatorExport_SetTextField($optionBreakDown[1], $enable, "parentScopeField");
+                //mayaUsdTranslatorExport_SetTextField($optionBreakDown[1], $enable, "parentScopeField");

```

Comment:
The reason I commented it out is that I dont think it make sense to fill the ParentScope section with the name of last export. This will also confuse the user since this field will be used as the default defaultPrim.

Reply:
I'll remove this `else if` and put a comment here?